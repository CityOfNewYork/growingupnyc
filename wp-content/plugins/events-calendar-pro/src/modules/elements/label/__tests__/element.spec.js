import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../element';

describe( 'Label', () => {
	test( 'should match snapshot', () => {
		const component = renderer.create(
			<Label className="test-class">
				Test label
			</Label>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
