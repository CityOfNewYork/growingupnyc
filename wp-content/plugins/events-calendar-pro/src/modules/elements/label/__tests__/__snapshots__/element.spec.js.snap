// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label should match snapshot 1`] = `
<div
  className="tribe-editor__events-pro__label test-class"
>
  <span
    className="tribe-editor__events-pro__label-text"
  >
    Test label
  </span>
</div>
`;
